src/System.Management.Automation/help/CommandHelpProvider.cs

```diff
@@ -572,7 +572,7 @@ private string FindHelpFile(CmdletInfo cmdletInfo)
                 // e.g., <AssemblyName>.ni.dll as supposed to <AssemblyName>.dll.
 
                 // When cmdlet metadata is generated for the 'HelpFile' field, we use the name assembly and we append '-Help.xml' to it.
-                // Because of this, if the cmdlet is part of an N�gen assembly, then 'HelpFile' field will be pointing to a help file which does not exist.
+                // Because of this, if the cmdlet is part of an N'gen assembly, then 'HelpFile' field will be pointing to a help file which does not exist.
```

Comment:
The original comment had a `0x92` byte, which is a curly apostrophe in CP-1252. I did wonder about whether it made sense, but I wanted to stay as close to the original as possible. I'll be happy to fix it to what it *should* be, though.